packages/motif/package.json

```diff
@@ -19,8 +19,8 @@
   ],
   "scripts": {
     "start": "vite --force --config vite.dev.config.ts",
-    "build": "npm run clean && tsc -b && vite build --config vite.prod.config.ts",
-    "transpile": "tsc --p ./tsconfig.json --noEmit true --strict --watch",
+    "build": "npm run clean && npx -p typescript tsc && vite build --config vite.prod.config.ts",
```

Comment:
Why do we need to install typescript here?

Reply:
I decided to reference the local package name while execute `tsc` with `npx -p typescript tsc` to ensure the typescript version are consistent across different environment.

The command ensure that we will always use the `typescript` install in our `node_modules` to perform bundling.